lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -772,19 +772,22 @@ def accessible_objects(user_or_attribute, operation, options = {})
       end
     end
 
-    # Version of accessible_objects which only returns objects that are
-    # ancestors of a specified root object or the object itself
-    def accessible_ancestor_objects(user_or_attribute, operation, root_object, options = {})
+    # Version of accessible_objects which only returns objects that are ancestors of a specified
+    # root object or the object itself. A set of policy element associations with the specified
+    # operation may be optionally provided.
+    def accessible_ancestor_objects(user_or_attribute, operation, root_object, associations_with_operation = nil, options: {})
```

Comment:
Would it be possible to make `associations_with_operation` a key in the options hash?

I think this would be ideal because we also wouldn't technically break backwards compatibility since we wouldn't have to keyword the options hash. 

Plus, it just feels a little more straightforward to have `associations_with_operation` identifier by a keyword when used.